README.md

```diff
@@ -194,6 +194,7 @@ botdarr:
 | EXISTING_ITEM_PATHS_BLACKLIST | If you want content to NOT appear in searches against your library, you can list blacklisted paths here in comma delimited form, and they will be ignored when building up responses | no |
 | MAX_DOWNLOADS_TO_SHOW | The max number of downloads to show. If you set this to any value less than or equal to 0, no downloads will show | yes | 20 |
 | MAX_RESULTS_TO_SHOW | The max number of results to show per search command. If you set this to any value less than 0, the bot won't startup | yes | 20 | 
+| NOTIFICATION_INTERVAL | The number of minutes between each notification. If you set this to any value less than 0, the bot won't startup | yes | 20 | 
```

Comment:
technically you are just logging the exception, so the bot will startup, but I don't really care :)

Reply:
Yeah, I wasn't sure about this. I tried to see where the MAX_DOWNLOADS_TO_SHOW and MAX_RESULTS_TO_SHOW would have thrown an error as well, but I didn't see anything specific. I didn't know if the delay function would have a problem with a negative, but I figured it was at least safer to say it wouldn't work, than to have it not work with no explanation.

Reply:
Oh, I don't know how we both missed this: the default value specified here says 20, but it should be 5, as that's what it is in the ApiRequests.java.